changelogs/fragments/9225-proxmox-module-refactoring.yml

```
@@ -0,0 +1,9 @@
+minor_changes:
+  - proxmox - refactors the proxmox module (https://github.com/ansible-collections/community.general/pull/9225)
+  - proxmox - fixes idempotency of template conversions (https://github.com/ansible-collections/community.general/pull/9225, https://github.com/ansible-collections/community.general/issues/8811).
```

Shouldn't tihs and the next fragments be moved to `bugfixes`?
```suggestion

bugfixes:
  - proxmox - fixes idempotency of template conversions (https://github.com/ansible-collections/community.general/pull/9225, https://github.com/ansible-collections/community.general/issues/8811).
```